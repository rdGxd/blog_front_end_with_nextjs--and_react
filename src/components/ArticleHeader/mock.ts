import { ArticleHeaderProps } from ".";
import { data } from "../../api/dados.json";

const { title, excerpt, cover, author, categories, createdAt } = data.posts.data[0].attributes;

const mockArticleHeader = {
  title,
  author,
  categories,
  cover,
  createdAt,
  excerpt,
} as ArticleHeaderProps;

export default mockArticleHeader;
