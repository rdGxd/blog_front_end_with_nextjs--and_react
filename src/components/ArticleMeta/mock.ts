import { ArticleMetaProps } from ".";
import { data } from "../../api/dados.json";

const { categories, author, createdAt } = data.posts.data[0].attributes;

const mockArticleMeta = {
  createdAt,
  categories: categories.data,
  author,
} as ArticleMetaProps;

export default mockArticleMeta;
