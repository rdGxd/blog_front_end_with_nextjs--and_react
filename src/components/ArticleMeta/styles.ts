import styled, { css } from "styled-components";

export const Wrapper = styled.div`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.small};
    color: ${theme.colors.darkerGray};
    font-style: italic;

    .categories span::after {
      content: ", ";
    }

    .categories span:last-child::after {
      content: "";
    }

    a:hover {
      filter: brightness(50%);
    }
  `}
`;
