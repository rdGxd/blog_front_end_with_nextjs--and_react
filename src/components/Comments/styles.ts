import styled, { css } from "styled-components";

export const Wrapper = styled.div`
  ${({ theme }) => css`
    max-width: ${theme.sizes.content};
    width: 100%;
    margin: ${theme.spacings.large} 0 auto;
    padding: 0 ${theme.spacings.large};
  `}
`;
