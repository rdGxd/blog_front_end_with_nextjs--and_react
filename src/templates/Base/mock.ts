import { BaseProps } from ".";
import { data } from "../../api/dados.json";

export default {
  settings: data.setting.data,
  children: "Ol√° mundo",
} as unknown as BaseProps;
