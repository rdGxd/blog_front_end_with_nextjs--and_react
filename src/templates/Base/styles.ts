import styled, { css } from "styled-components";

export const Wrapper = styled.div``;

export const HeaderContainer = styled.header`
  ${({ theme }) => css`
    ${theme.spacings.xhuge}
  `}
`;

export const ContentContainer = styled.div`
  ${({ theme }) => css`
    max-width: 120rem;
    width: 100%;
    margin: 0 auto;
    margin-bottom: ${theme.spacings.xhuge};
  `}
`;

export const FooterContainer = styled.footer`
  ${({ theme }) => css`
    max-width: 120rem;
    width: 100%;
    margin: 0 auto;
    padding: ${theme.spacings.large};
  `}
`;
