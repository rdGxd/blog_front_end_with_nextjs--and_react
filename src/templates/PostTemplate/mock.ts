import { PostTemplateProps } from ".";
import { data } from "../../api/dados.json";

export default {
  settings: data.setting,
  post: data.posts.data[0].attributes,
} as unknown as PostTemplateProps;
