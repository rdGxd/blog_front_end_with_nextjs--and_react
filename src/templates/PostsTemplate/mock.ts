import { PostsTemplateProps } from ".";
import { data } from "../../api/dados.json";

export default {
  settings: data.setting,
  posts: { data: data.posts.data },
} as unknown as PostsTemplateProps;
