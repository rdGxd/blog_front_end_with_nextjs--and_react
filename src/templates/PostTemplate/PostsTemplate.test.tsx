import { PostTemplate } from ".";
import { renderTheme } from "../../styles/render-theme";
import mock from "./mock";

describe("<PostTemplate />", () => {
  it("should match snapshot", () => {
    const { container } = renderTheme(<PostTemplate {...mock} />);

    expect(container).toMatchSnapshot();
  });
});
