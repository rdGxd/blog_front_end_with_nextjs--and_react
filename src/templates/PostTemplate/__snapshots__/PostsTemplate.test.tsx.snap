// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<PostTemplate /> > should match snapshot 1`] = `
.c19 {
  font-size: 1.6rem;
  color: #AAAAAA;
  font-style: italic;
}

.c19 .categories span::after {
  content: ", ";
}

.c19 .categories span:last-child::after {
  content: "";
}

.c19 a:hover {
  filter: brightness(50%);
}

.c6 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c12 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: none;
}

.c16 {
  color: #000000;
  line-height: 1.2;
  font-size: 6.4rem;
  text-transform: none;
}

.c15 {
  padding-bottom: 4.0rem;
  margin-bottom: 4.0rem;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c15 .c5 {
  margin: 0;
  margin-bottom: 2.4rem;
}

.c17 {
  margin: 2.4rem 0;
  font-size: 2.4rem;
}

.c18 {
  max-width: 100%;
  display: block;
  margin-bottom: 2.4rem;
}

.c22 {
  font-size: 2.4rem;
}

.c22 p {
  margin: 2.4rem 0;
}

.c22 a:visited,
.c22 a:link {
  color: #dc143c;
  text-decoration: none;
  transition: all 300ms ease-in-out;
}

.c22 a:hover {
  filter: brightness(80%);
}

.c22 code {
  font-family: monospace;
  color: #dc143c;
  font-size: 1.6rem;
  background: #DDDDDD;
  padding: 0.2rem;
  margin: 0.2rem;
}

.c22 pre {
  background: #000000;
  padding: 2.4rem;
  font-family: monospace;
  color: #FFFFFF;
  margin: 2.4rem 0;
  width: 100%;
  overflow-x: auto;
  font-size: 1.6rem;
}

.c22 pre code {
  color: inherit;
  background: inherit;
}

.c22 img {
  max-width: 100%;
}

.c22 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c22 .image figcaption {
  text-align: center;
  font-size: 1.6rem;
  padding: 1.6rem;
  line-height: 1.3;
}

.c22 .image-style-side {
  max-width: 50%;
  float: right;
  margin: 2.4rem 0;
}

.c22 hr {
  border: none;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c22 ul,
.c22 ol {
  margin: 2.4rem 4.0rem;
}

.c22 .table {
  width: 100%;
  border-collapse: collapse;
  margin: 2.4rem 0;
}

.c22 table {
  width: 100%;
  border-collapse: collapse;
}

.c22 table td,
.c22 table th {
  padding: 1.6rem;
  border: 0.1rem solid #DDDDDD;
}

.c22 blockquote {
  border-left: 0.5rem solid #dc143c;
  color: #AAAAAA;
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c14 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 96rem;
}

.c20 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 80rem;
}

.c24 {
  margin: 2.4rem 0;
}

.c24 span {
  margin: 0 0rem 0 0.5rem;
}

.c24 span::before {
  content: " ";
}

.c24 span::after {
  content: ", ";
}

.c24 span:last-child::after {
  content: "";
}

.c24 a:hover {
  filter: brightness(80%);
}

.c26 {
  text-align: center;
  border-top: 0.1rem solid #DDDDDD;
}

.c26 a {
  color: inherit;
  text-decoration: none;
}

.c26 .c21 {
  font-size: 1.6rem;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c27 {
  position: fixed;
  background: #000000;
  color: #FFFFFF;
  display: flex;
  align-items: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
  opacity: 0.7;
}

.c7 {
  display: flex;
  align-items: center;
  text-decoration: none;
  color: inherit;
}

.c7 img {
  width: 15rem;
  height: 15rem;
  border-radius: 50%;
}

.c10 {
  display: flex;
  align-items: center;
  justify-content: center;
  padding-bottom: 6.4rem;
  width: 100%;
  max-width: 96rem;
  color: #333333;
  font-size: 1.6rem;
  margin: 0 auto;
}

.c10 .c5 {
  margin: 0 0 calc(1.6rem -1rem);
}

.c11 {
  display: flex;
  flex-flow: column wrap;
  margin-left: 3.2rem;
  max-width: 48rem;
}

.c8 {
  display: block;
  color: #FFFFFF;
  text-decoration: none;
  margin-bottom: 1.6rem;
  font-size: 1.8rem;
  border-right: 0.5rem solid #000000;
  transition: all 300ms ease-in-out;
}

.c8:hover {
  border-right: 0.5rem solid #dc143c;
  color: #dc143c;
}

.c2 {
  display: flex;
  background: #000000;
  padding: 3.2rem;
  position: fixed;
  z-index: 1;
  width: 100%;
  max-width: 32rem;
  height: 100vh;
  top: 0;
  left: 0;
  transition: all 300ms ease-in-out;
  overflow-y: auto;
  left: -35rem;
  overflow-y: hidden;
}

.c3 {
  margin: auto;
  width: 100%;
}

.c4 .c5 {
  display: flex;
  justify-content: center;
  margin: 0;
  margin-bottom: 4.0rem;
}

.c4 .c5 img {
  border: 0.5rem solid #dc143c;
}

.c0 {
  position: fixed;
  top: 2.4rem;
  color: #FFFFFF;
  background: #000000;
  z-index: 2;
  width: 3rem;
  height: 3rem;
  left: 26rem;
  left: 1rem;
  color: #FFFFFF;
}

.c9 {
  margin-top: 6.4rem;
  padding: 0 3.2rem;
}

.c13 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  margin-bottom: 6.4rem;
}

.c25 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  padding: 3.2rem;
}

.c23 {
  padding: 0 3.2rem;
}

@media (max-width: 768px) {
  .c16 {
    font-size: 4.0rem;
  }
}

@media (max-width: 768px) {
  .c22 {
    font-size: 2rem;
  }

  .c22 .image-style-side {
    max-width: 100%;
    float: none;
    margin: 0;
  }
}

@media (max-width: 360px) {
  .c10 {
    display: flex;
    flex-flow: column wrap;
  }

  .c10>.c5 {
    margin: 0 0 2.4rem 0;
  }
}

@media (max-width: 360px) {
  .c11 {
    margin-left: 0;
  }
}

@media (max-width: 768px) {
  .c2 {
    left: -32rem;
  }
}

@media (max-width: 768px) {
  .c0 {
    left: -0.1rem;
  }
}

<div>
  <div
    class=""
  >
    <a
      aria-label="Open or close menu"
      class="c0"
      href="#"
      title="Open or close menu"
    >
      <svg
        aria-hidden="true"
        aria-label="Open menu"
        class="c1"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </a>
    <div
      aria-hidden="true"
      class="c2"
    >
      <nav
        class="c3"
      >
        <div
          class="c4"
        >
          <h1
            class="c5 c6"
          >
            <a
              class="c7"
              href="/"
              target="_self"
            >
              <img
                alt="Rodrigo Silva"
                src="https://res.cloudinary.com/dccldzodg/image/upload/v1700489959/1325363_6dfc0b7add.png"
              />
            </a>
          </h1>
        </div>
        <a
          class="c8"
          href="TESTE"
          target="_self"
        >
          #TESTE
        </a>
      </nav>
    </div>
    <header
      class="c9"
    >
      <header
        class="c10"
      >
        <h1
          class="c5 c6"
        >
          <a
            class="c7"
            href="/"
            target="_self"
          >
            <img
              alt="Rodrigo Silva - Blog do Rodrigo"
              src="https://res.cloudinary.com/dccldzodg/image/upload/v1700489959/1325363_6dfc0b7add.png"
            />
          </a>
        </h1>
        <div
          class="c11"
        >
          <h2
            class="c5 c12"
          >
            Rodrigo Silva
          </h2>
          <p>
            Blog do Rodrigo
          </p>
        </div>
      </header>
    </header>
    <div
      class="c13"
    >
      <article
        class=""
      >
        <div
          class="c14"
        >
          <header
            class="c15"
          >
            <h1
              class="c5 c16"
            >
              6PAPAPA
            </h1>
            <p
              class="c17"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            </p>
            <img
              alt="6PAPAPA"
              class="c18"
              src="https://res.cloudinary.com/dccldzodg/image/upload/v1700490199/360_F_571067620_JS_5_T5_Tk_Dtu3gf8_Wqm78_Ko_JRF_1vob_Pvo6_1dff1a52c6.jpg"
            />
            <div
              class="c19"
            >
              <p>
                <span>
                  Por 
                </span>
                <a
                  href="/author/rodrigo-silva"
                >
                  Rodrigo Silva
                </a>
                <span
                  class="separator"
                >
                   | 
                </span>
                <time
                  datetime="2023-11-20T14:26:29.353Z"
                >
                  20 de nov. de 2023
                </time>
              </p>
            </div>
          </header>
        </div>
        <div
          class="c20"
        >
          <div
            class="c21 c22"
          >
            Nostro 
            <strong>
              definitiones
            </strong>
             (Tech) sed cu - Jo√£o. Sea Object Destructuring, useState dicat fabulas 
            <a
              href="\\\\\\"https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad\\\\\\""
            >
              consetetur
            </a>
             no, odio deseruisse assueverit mei eu, vis probo nonumes molestie id. Ipsum movet liberavisse an mea, diceret vituperata ne his, delenit apeirian quo ea. Imperdiet percipitur te eos. Per ludus aliquam cu. 
            <a
              href="\\\\\\"https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad\\\\\\""
            >
              Eos ex tincidunt necessitatibus, timeam theophrastus mel cu
            </a>
            .
            <p />
            <pre>
              <code
                class="\\\\\\"language-plaintext\\\\\\""
              >
                import { useEffect, useRef, useState } from 'react';\\n\\nconst useMyHook = (cb, delay = 1000) =&gt; {\\n  const savedCb = useRef();\\n\\n  useEffect(() =&gt; {\\n    savedCb.current = cb;\\n  }, [cb]);\\n\\n  useEffect(() =&gt; {\\n    const interval = setInterval(() =&gt; {\\n      savedCb.current();\\n    }, delay);\\n\\n    return () =&gt; clearInterval(interval);\\n  }, [delay]);\\n};
              </code>
            </pre>
            <p>
              Qui eu 
              <i>
                mutat
              </i>
               exerci 
              <span
                style="\\\\\\"background-color:hsl(60,75%,60%);\\\\\\""
              >
                dolores
              </span>
              , his no 
              <span
                style="\\\\\\"color:hsl(270,75%,60%);\\\\\\""
              >
                repudiare
              </span>
               constituam 
              <s>
                accommodare
              </s>
              , vis ea sint 
              <u>
                tempor
              </u>
               eirmod. In mei eruditi moderatius, ex per iudico essent repudiandae. In vis vidisse menandri, an has repudiare disputationi delicatissimi. In has wisi justo postulant. Ei viderer ornatus mel, usu an partiendo tincidunt, ea eum brute ullamcorper.
            </p>
            <p>
              10
              <sup>
                2
              </sup>
              *5
              <sup>
                2
              </sup>
               - H
              <sub>
                2
              </sub>
              0
            </p>
            <p
              style="\\\\\\"text-align:center;\\\\\\""
            >
              <strong>
                Center: 
              </strong>
              Abhorreant reformidans eum an, per equidem repudiandae no, ex cum elitr argumentum persequeris. In eam consul theophrastus delicatissimi, lobortis rationibus efficiendi mea ei, nam natum dolore scripserit ea. Sit partiendo eloquentiam ne. Sea no autem dolores, ridens gloriatur mel cu. Munere nullam epicuri his id, dicam definiebas deterruisset eu vim. Ea veritus deleniti usu, nam ei idque choro volumus, ex nisl aliquam bonorum eam.
            </p>
            <p>
              Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.
            </p>
            <p
              style="\\\\\\"text-align:right;\\\\\\""
            >
              <strong>
                Right:
              </strong>
               Mel in lorem feugiat, tation dolore et eum. Id has utamur albucius molestiae. Dico justo maiestatis ei mei, labitur laboramus voluptatum nam ut. Saperet propriae nec an.
            </p>
            <p>
              Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit. In persius explicari conceptam nec, in populo ponderum consequat vis. Gloriatur conclusionemque no qui, vitae nominati nec te, laoreet torquatos accommodare ex qui. Eu natum mediocritatem eos, cum ex alia oportere.
            </p>
            <p
              style="\\\\\\"text-align:justify;\\\\\\""
            >
              <strong>
                Justify:
              </strong>
               Cu vis nonumes petentium disputationi, invidunt aliquando ex ius, ocurreret suscipiantur conclusionemque eu qui. Probo nemore ut est. Et eam dicta oratio eligendi. Quem dicta dolore vel et, at reque gloriatur definitionem sed.
            </p>
            <p>
              Nec simul ullamcorper at. Mea possit dolores persecuti te, ei quidam apeirian quo. Has quas corpora ne. Ius ad malis iusto delenit, sit ferri idque ex.
            </p>
            <blockquote>
              <p>
                Blockquote: Et sed legere rationibus, vitae evertitur ad eam. Eu amet vero officiis vis. Ad nullam pericula neglegentur sit, vix ei regione legendos argumentum. Mei an quas convenire, nobis dicunt mei ut. Pro no debet referrentur, ei est falli libris. Eu per summo salutandi hendrerit, salutatus democritum interesset nec ut. Wisi lobortis incorrupte ei mea, explicari interpretaris has et.
              </p>
            </blockquote>
            <p>
              A new line:
            </p>
            <hr />
            <p>
              Now keep going‚Ä¶
            </p>
            <figure
              class="\\\\\\"image\\\\\\""
            >
              <a
                href="\\\\\\"https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad\\\\\\""
              >
                <img
                  alt="\\\\\\"O"
                  da=""
                  imagem\\"=""
                  src="\\\\\\"https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg\\\\\\""
                  text=""
                />
              </a>
              <figcaption>
                Aqui est√° uma imagem
              </figcaption>
            </figure>
            <p>
              Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
            </p>
            <figure
              class="\\\\\\"image"
              image-style-side\\"=""
            >
              <img
                alt="\\\\\\"Uma"
                direita\\"=""
                imagem=""
                src="\\\\\\"https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg\\\\\\""
                √†=""
              />
              <figcaption>
                Uma imagem √† direita
              </figcaption>
            </figure>
            <p>
              Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
            </p>
            <p>
              Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
            </p>
            <p>
              Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
            </p>
            <p>
              Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
            </p>
            <p>
              Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
            </p>
            <p>
              Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
            </p>
            <p>
              Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
            </p>
            <p>
              Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
            </p>
            <p>
              Bullet:
            </p>
            <ul>
              <li>
                Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
              </li>
              <li>
                iriure posidonium consectetuer ad eam.
              </li>
              <li>
                Nobis reprimique duo in, eam corpora mandamus
              </li>
              <li>
                ex. In
              </li>
              <li>
                quod euismod
              </li>
              <li>
                eum, ne habemus cotidieque necessitatibus cum.
              </li>
              <li>
                Liber recteque duo no, ludus reprehendunt has an.
              </li>
            </ul>
            <p>
              Now ordered:
            </p>
            <ol>
              <li>
                Item 1
              </li>
              <li>
                Item
              </li>
              <li>
                Item
              </li>
              <li>
                Item
              </li>
            </ol>
            <p>
              A think that's it‚Ä¶
            </p>
            <figure
              class="\\\\\\"table\\\\\\""
            >
              <table>
                <thead>
                  <tr>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                    <th>
                      Title 1
                    </th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                  </tr>
                  <tr>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                  </tr>
                  <tr>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                    <td>
                      Item
                    </td>
                  </tr>
                </tbody>
              </table>
            </figure>
            <p>
              A video:
            </p>
            <figure
              class="\\\\\\"media\\\\\\""
            >
              <oembed
                url="\\\\\\"https://www.youtube.com/watch?v=oLmRV25eQvo\\\\\\""
              />
            </figure>
            <p>
              The end.
            </p>
          </div>
        </div>
      </article>
      <div
        class="c23"
      >
        <p
          class="c24"
        >
          tags:
           
          <span>
            <a
              href="/tag/bc"
            >
              BC
            </a>
          </span>
          <span>
            <a
              href="/tag/bc2"
            >
              BC2
            </a>
          </span>
        </p>
      </div>
    </div>
    <footer
      class="c25"
    >
      <footer
        class="c26"
      >
        <div
          class="c21 c22"
        >
          texto do footer
        </div>
      </footer>
    </footer>
    <a
      aria-label="Go to top"
      class="c27"
      href="#"
      title="Go to top"
    >
      <svg
        aria-hidden="true"
        class="c1"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M7.41 15.41 12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
        />
      </svg>
    </a>
  </div>
</div>
`;
